### ISSUE TITLE
Narrator does not convey error information when no more Right Parenthesis can be added in expression.

### ISSUE THREAD

**Issue by MicrosoftIssueBot** [Microsoft/calculator#282]
Test Environment:
OS Version: Windows 10 Build 17133.1
App Name: Calculator
App Version: 10.1804.911.0

Repro Steps: 

Launch "Calculator" app.
Launch Narrator.
Navigate to “Programmer/Scientific" control in the navigation pane and activate it.
Enter a number in the display.
Navigate to "Left Parenthesis" button in “Programmer/Scientific Calculator" window and activate it five times.
Now, navigate to "Right Parenthesis" button and activate it five times.
Try to activate "Right Parenthesis" button for the sixth time. No more Right Parenthesis can be added in expression now since the count of Left Parenthesis and Right Parenthesis is equal.
Verify if Narrator conveys error information when no more Right Parenthesis can be added in expression.
Actual Result:

Narrator does not convey error information when no more Right Parenthesis can be added in expression once the count of Left Parenthesis and Right Parenthesis is equal.       

Expected Result:

Narrator should convey error information when no more Right Parenthesis can be added in expression.

**Comment by MicrosoftIssueBot** [comment]
This is your friendly Microsoft Issue Bot. I created this issue automatically as requested by a team member.

**Comment by mdtauk** [comment]
With XAML Textfield validation coming in 19H1 - should there eventually be warnings when someone tries to evaluate with a missing closing bracket/parenthesis?

What about other textfield validation and errors, like incorrect pasting of values?

**Comment by grochocki** [comment]
> With XAML Textfield validation coming in 19H1 - should there eventually be warnings when someone tries to evaluate with a missing closing bracket/parenthesis?

We should definitely evaluate it as an option, though we are fairly forgiving (e.g., "(12+4" evaluates to 16). This particular fix is more about communicating state for screen readers as opposed to any data validation.
